test/inputs/Multi-year Investments/graph-flows-data.csv

```diff
@@ -0,0 +1,7 @@
+asset_name,asset_name,
+from_asset,to_asset,carrier
```

Comment:
Maybe we need to add `investment_method` to flows (for transport investment), otherwise what formulation do we want to use for flows? @datejada @g-moralesespana

Reply:
Given the reply from German on Teams, indeed we will only use simple method for transport investment. We can either leave it empty, meaning we will use simple _by default_. Or we can add `investment_method` here and specify simple here. @abelsiqueira What do you think?